### ISSUE TITLE
DPFManager 3.5.1 Lite 64-bit Windows won't run

### ISSUE THREAD

**Issue by pmay** [EasyinnovaSL/DPFManager#493]
DPFManager-lite installs ok (selecting default options) but won't run as it fails to find "C:\Program Files\DPF Manager\runtime\bin\server\jvm.dll".

![dpfmanager-3 5 1-lite-windows-64_err1](https://user-images.githubusercontent.com/1150504/39466520-5eeeae92-4d21-11e8-897b-06083dcbb5e4.JPG)

I have JDK1.8.0 installed and %JDK1.8.0%\bin in my path settings.
I note that there is no "runtime" directory at all in my DPFManager installation.

The DPFManager version with the JVM installs and runs without this warning.

**Comment by victormunoz** [comment]
Yes, it seems the exe is not working correctly in the lite version.
However you can still run the lite version with the command `java -jar "app/DPF Manager-jfx.jar"`
